.github/dependabot.yml

```diff
@@ -9,7 +9,27 @@ updates:
     directory: "/" # Location of package manifests
     schedule:
       interval: "daily"
+    ignore:
+      - dependency-name: "github.com/ava-labs/avalanchego" # manually updated
+      - dependency-name: "github.com/ava-labs/libevm" # manually updated
+      - dependency-name: "github.com/VictoriaMetrics/fastcache" # geth dependency
+      - dependency-name: "github.com/hashicorp/go-bexpr" # geth dependency
+      - dependency-name: "github.com/holiman/*" # geth dependency
+      - dependency-name: "github.com/holiman/billy" # geth dependency
+      - dependency-name: "github.com/holiman/bloomfilter/v2" # geth dependency
+      - dependency-name: "github.com/holiman/uint256" # geth dependency
+      - dependency-name: "github.com/mattn/go-colorable" # geth dependency
+      - dependency-name: "github.com/mattn/go-isatty" # geth dependency
+      - dependency-name: "github.com/tyler-smith/go-bip39" # geth dependency
+      - dependency-name: "github.com/urfave/cli/v2" # geth dependency
+      - dependency-name: "golang.org/x/time" # geth dependency
+      - dependency-name: "gopkg.in/natefinch/lumberjack.v2" # geth dependency
   - package-ecosystem: github-actions
     directory: "/"
     schedule:
       interval: weekly
+    ignore:
+      - dependency-name: "ava-labs/avalanchego" # manually updated
```

Comment:
Just out of curiousity, what Github actions are sourced from AvalancheGo that dependabot picks up on?

Reply:
For now it's only `ava-labs/avalanchego/.github/actions/run-monitored-tmpnet-cmd` which is enforced by CI to match the Go.mod avalanchego version. However, it is true we could add more actions that don't need to match the go.mod avalanchego version further down the line, so I changed this to be an exact match `ava-labs/avalanchego/.github/actions/run-monitored-tmpnet-cmd`